frontend/src/app.vue

```diff
@@ -321,4 +321,9 @@ export default app;
 <style lang="scss">
 @import './styles/style.scss';
 @import './styles/panels.scss';
+
+.hljs-comment,
+.hljs-meta {
+  color: #5d635a;
+}
```

Comment:
Will it be better if this css is moved closer to the element like in `v-authorship.vue`?

Reply:
Actually do you think it is a good idea to just create our own css file and import it instead of doing it partially over here? In case in future updates this theme is removed or we want more configurations?

Reply:
Sure